### ISSUE TITLE
updates: ProgrammingError: syntax error at or near ")"

### ISSUE THREAD

**Issue by psegedy** [RedHatInsights/vmaas#124]
I've found 2 packages so far that result in 500 Internal Server Error
```
Red_Hat_Enterprise_Linux-Release_Notes-7-en-US-0:7-2.el7.noarch
crontabs-1.11-6.20121102git.el7.noarch

# curl http://localhost:8080/api/v1/updates/crontabs-1.11-6.20121102git.el7.noarch               
<html><title>500: Internal Server Error</title><body>500: Internal Server Error</body></html>%
# curl http://localhost:8080/api/v1/updates/Red_Hat_Enterprise_Linux-Release_Notes-7-en-US-0:7-2.el7.noarch
<html><title>500: Internal Server Error</title><body>500: Internal Server Error</body></html>%    
```
Server side error:
```
vmaas-webapp | ERROR:tornado.application:Uncaught exception GET /api/v1/updates/Red_Hat_Enterprise_Linux-Release_Notes-7-en-US-0:7-2.el7.noarch (172.18.0.1)
vmaas-webapp | HTTPServerRequest(protocol='http', host='localhost:8080', method='GET', uri='/api/v1/updates/Red_Hat_Enterprise_Linux-Release_Notes-7-en-US-0:7-2.el7.noarch', version='HTTP/1.1', remote_ip='172.18.0.1', headers={'Host': 'localhost:8080', 'Accept': '*/*', 'User-Agent': 'curl/7.55.1'})
vmaas-webapp | Traceback (most recent call last):
vmaas-webapp |   File "/usr/lib64/python2.7/site-packages/tornado/web.py", line 1412, in _execute
vmaas-webapp |     result = method(*self.path_args, **self.path_kwargs)
vmaas-webapp |   File "/app/app.py", line 31, in get
vmaas-webapp |     response = self.process_string(name)
vmaas-webapp |   File "/app/app.py", line 54, in process_string
vmaas-webapp |     return self.application.updatesapi.process_list({'package_list': [data]})
vmaas-webapp |   File "/app/updates.py", line 209, in process_list
vmaas-webapp |     self.cursor.execute("select pkg_id, repo_id from pkg_repo where pkg_id in %s;", [tuple(update_pkg_ids)])
vmaas-webapp | ProgrammingError: syntax error at or near ")"
vmaas-webapp | LINE 1: select pkg_id, repo_id from pkg_repo where pkg_id in ();
vmaas-webapp |                                                               ^
vmaas-webapp | 
vmaas-webapp | ERROR:tornado.access:500 GET /api/v1/updates/Red_Hat_Enterprise_Linux-Release_Notes-7-en-US-0:7-2.el7.noarch (172.18.0.1) 61.20ms
```
Which also blocks transaction as mentioned in https://github.com/RedHatInsights/vmaas/issues/67

**Comment by ggainey** [comment]
Happens when the requested package isn't in the DB

**Comment by psegedy** [comment]
Actually I'm able to find these packages in the DB. Am I missing something?
```
vmaas=> select * from package where name='crontabs';
  id   |   name   | evr_id | arch_id |                 checksum                 | checksum_type_id 
-------+----------+--------+---------+------------------------------------------+------------------
  2183 | crontabs |   1157 |       3 | a37c6c3024b08f25425d81ea8d47ed6300f0bdfd |                1
 14485 | crontabs |    912 |       3 | 60196387bd32d6afb60db0e4b1b0542883deaca7 |                1
 49936 | crontabs |  10926 |       3 | 15320df538a2a6797fb0b72fa1a59543c513c479 |                1
(3 rows)



 id   |                      name                      | evr_id | arch_id |                 checksum                 | checksum_type_id 
-------+------------------------------------------------+--------+---------+------------------------------------------+------------------
 35597 | Red_Hat_Enterprise_Linux-Release_Notes-7-en-US |  10985 |       3 | cca55a5fd84d07f96bef2ff06c5fcd834b4a0ecf |                1
 50864 | Red_Hat_Enterprise_Linux-Release_Notes-7-en-US |  12545 |       3 | 9c4533f7004bc69a48abbfd2c5a16602300ac765 |                1
 52636 | Red_Hat_Enterprise_Linux-Release_Notes-7-en-US |   7075 |       3 | 43b36ba0ac584d17bdcf3dba60eb7db6eaec2e92 |                1
(3 rows)

```

**Comment by grinrag** [comment]
The problem also could happen if we no packages with greater EVR, or no suitable errata. Fixed in PR #134

**Comment by jdobes** [comment]
#124 fixes this, for `Red_Hat_Enterprise_Linux-Release_Notes-7-en-US-0:7-2.el7.noarch` and `crontabs-1.11-6.20121102git.el7.noarch` is empty result expected because these are already latest versions.
